src/components/Navbar.tsx

```diff
@@ -0,0 +1,84 @@
+import { useEffect, useState } from "react";
+
+const useScrollPosition = () => {
+	const [scrollPos, setScrollPos] = useState(0);
+
+	const handleScroll = () => {
+		setScrollPos(window.scrollY);
+	};
+
+	useEffect(() => {
+		window.addEventListener("scroll", handleScroll);
+		return () => window.removeEventListener("scroll", handleScroll);
+	}, []);
+
+	return scrollPos;
+};
+
+const Navbar = () => {
+	const scrollPos = useScrollPosition();
+	const bannerOpacity = Math.max(1 - scrollPos / 100, 0);
+	return (
+		<>
```

Comment:
Any reason for having the fragment here? Everything is wrapped in the `nav` tag so the fragment is not needed.

Reply:
Fixed in latest commit